docker/BUILD.bazel

```diff
@@ -75,6 +75,16 @@ genrule(
     visibility = ["//visibility:private"],
 )
 
+genrule(
+    name = "pace_tag",
+    srcs = ["PACE"],
+    outs = ["docker_pace_tag"],
+    cmd = """
+    $$(tr -d '\n' < $(execpath PACE)) > $@
```

Comment:
You can remove the nested evaluation (`$$()`):
```bash
tr -d '\n' < $(execpath PACE) > $@
```
You can also replace `$(execpath PACE)` with `$<`:
```bash
tr -d '\n' < $< > $@